cypress/utils/modules/StaffPage/StaffDetailsTestFunctions.ts

```diff
@@ -30,92 +26,50 @@ export class StaffDetailsTestFunctions {
     this.getDetailsTileTutorial().contains('Konsultacje');
     this.getDetailsTileTutorialSchedule().should('exist');
-    this.getDetailsTileTutorialSchedule().contains('Czwartki 8-10 ');
+    this.getDetailsTileTutorialSchedule().contains('Środy, w godzinach 10:15-11:45 lub po umówieniu się.');
   };
-
-  // commented out because language option will be change later
-  //   testDetailsWithoutTutorialEN = () => {
-  //     cy.visit('/api/staff/63cb1cf20ada513d831bc83d');
-  //     this.mockGETDetailsWithoutTutorial();
-  //     LanguageChange.changeLanguage();
-  //     this.getStaffDetailsName().should('exist');
-  //     this.getDetailsTile().should('exist');
-  //     this.getDetailsTile().should('have.length', 4);
-  //     this.getDetailsTile().eq(1).contains('Wydział Matematyki, Fizyki i Informatyki');
-  //     this.getDetailsTile().eq(2).contains('Instytut Informatyki');
-  //     this.getDetailsTile().eq(3).contains('Zakład Sztucznej Inteligencji');
-  //     this.getDetailsTileContact().should('exist');
-  //     this.getDetailsTileContact().contains('Contact');
-  //     this.getDetailsTileContactEmail().contains('E-mail: joanna.czarnowska@mat.ug.edu.pl');
-  //     this.getDetailsTileTutorial().should('not.exist');
-  //     this.getDetailsTileTutorialSchedule().should('not.exist');
-  //   };
-
-  //   // commented out because language option will be change later
-  //   testDetailsWithTutorialEN = () => {
-  //     cy.visit('/api/staff/63cb1cf20ada513d831bc8c9');
-  //     this.mockGETStaffDetailsWithTutorial();
-  //     LanguageChange.changeLanguage();
-  //     this.getStaffDetailsName().should('exist');
-  //     this.getDetailsTile().should('exist');
-  //     this.getDetailsTile().should('have.length', 4);
-  //     this.getDetailsTilePost().contains('Asystent');
-  //     this.getDetailsTile().eq(1).contains('Wydział Matematyki, Fizyki i Informatyki');
-  //     this.getDetailsTile().eq(2).contains('Instytut Informatyki');
-  //     this.getDetailsTile().eq(3).contains('Zakład Optymalizacji Kombinatorycznej');
-  //     this.getDetailsTileContact().should('exist');
-  //     this.getDetailsTileContact().contains('Contact');
-  //     this.getDetailsTileContactEmail().contains('E-mail: mateusz.miotk@inf.ug.edu.pl');
-  //     this.getDetailsTileTutorial().should('exist');
-  //     this.getDetailsTileTutorial().contains('Tutorial');
-  //     this.getDetailsTileTutorialSchedule().should('exist');
-  //     this.getDetailsTileTutorialSchedule().contains('Czwartki 8-10 ');
-  //   };
-
   // pending status and error handling
 
   private getSkeletonRow = () => cy.getBySelector('skeleton-row');
   private getSkeletonTile = () => cy.getBySelector('skeleton-tile');
 
   testDetailsContentPendingStatus = () => {
-    cy.intercept('GET', '/staff/63cb1cf20ada513d831bc83d', request => {
+    cy.intercept('GET', 'https://api-kiosk-dev.onrender.com/staff/656a74c983a231f468534835', request => {
```

Comment:
maybe it would be a good idea to create a constant with api url? it would be easier to change tests in case we change api's url in the future